d2bs/kolbot/libs/bots/BoBot.js

```diff
@@ -0,0 +1,218 @@
+/**
+*	@filename	BoBot.js
+*	@author		casualBotter
+*	@desc		Meant to run alongside Enchant to Bo players
+*/
+
+function BoBot(){
+	var command, hostile, nick, spot, tick, s, m,
+		startTime = getTickCount(),
+		shitList = [],
+		greet = [];
+
+		this.Bo = function (nick) {
+		if (!Misc.inMyParty(nick)) {
+			say("Accept party invite, noob.");
+
+			return false;
+		}
+
+		var partyUnit,
+			unit = getUnit(0, nick);
+
+		if (!unit) {

```

Comment:
Don't use `getUnit()` here, if it succeeds we jump to line 46 and barb will precast in town.

Reply:
What do I use? I tried taking out that part but I got errors